package.json

```diff
@@ -45,30 +45,36 @@
     "watch:test": "npm test -- --auto-watch --no-single-run"
   },
   "dependencies": {
-    "@angular/common": "2.0.0-rc.6",
-    "@angular/compiler": "2.0.0-rc.6",
+    "@angular/common": "2.0.0",
```

Comment:
Why are you upgrading deps in this PR? Is it necessary to make tests run, i.e. is there a bug in 2.0.0-rc.6 that prevents tests from passing?